From 466289ab50f5cc0bc2a434f6c94f7caaf98c11c8 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:01 -0400
Subject: [redhat] kabi: Adding symbol scsi_host_put (drivers/scsi/hosts.c)

Message-id: <1a2f7f35d07450b67fd61c721b174322cbec17cc.1410510773.git.poros@redhat.com>
Patchwork-id: 96132
O-Subject: [PATCH RHEL7.1 181/233] kabi: Adding symbol scsi_host_put (drivers/scsi/hosts.c)
Bugzilla: 1071099
RH-Acked-by: Rob Evers <revers@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Decrement a Scsi_Host ref count.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/scsi_host_put
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/scsi_host_put
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/scsi_host_put

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/scsi_host_put b/redhat/kabi/kabi-module/kabi_ppc64/scsi_host_put
new file mode 100644
index 0000000..603dcca
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/scsi_host_put
@@ -0,0 +1,2 @@
+#1-
+0xd5ac345b scsi_host_put vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/scsi_host_put b/redhat/kabi/kabi-module/kabi_s390x/scsi_host_put
new file mode 100644
index 0000000..1f2a554
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/scsi_host_put
@@ -0,0 +1,2 @@
+#1-
+0x55781169 scsi_host_put vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/scsi_host_put b/redhat/kabi/kabi-module/kabi_x86_64/scsi_host_put
new file mode 100644
index 0000000..4c6b2ff
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/scsi_host_put
@@ -0,0 +1,2 @@
+#1-
+0x6f8d91ba scsi_host_put vmlinux EXPORT_SYMBOL
-- 
1.7.1